src/driver/mongodb/typings.ts

```diff
@@ -1,5 +1,4 @@
-import {EventEmitter} from "events";
-import {Readable, Writable} from "stream";
+import {EventEmitterWrapper, StreamReadableWrapper, StreamWritableWrapper} from "../../platform/PlatformTools";
```

Comment:
do we have alternative to create such "wrappers"? Maybe we simply can define for browser package additional typings, like:

```ts
declare module "stream";
declare module "events";
```

Would this approach work?

Reply:
I will give it a try

Reply:
I don't know where I would define these module so I commited a version where the normal PlatformTools just exports the classes from `stream` and `event` and the browser version an empty classes for it. This would make it less of a change